requirements-test.txt

```diff
@@ -6,6 +6,7 @@ grpcio==1.15.0
 Jinja2==2.10
 mock==2.0.0
 pybadges==1.0.2
+pymysql==0.9.3
```

Comment:
you can probably remove unused dependencies like bigquery

Reply:
Keep it for now, will remove it once the changes are released to PyPI (then picked up by our dashboard builder and related stuff).

Reply:
SGTM